corehq/ex-submodules/pillowtop/utils.py

```diff
@@ -356,6 +356,7 @@ def get_errors_with_ids(es_action_errors):
     return [
         (item['_id'], item.get('error'))
         for op_type, item in _changes_to_list(es_action_errors)
+        if not (item.get('op_type') == 'delete' and item.get('status') == 404)
```

Comment:
Looks like this should have been 

```
if not (op_type == 'delete' and item.get('status') == 404)
```
and that that's the cause of most of the pillow errors on prod